packages/material/scss/scheduler/_layout.scss

```diff
@@ -145,8 +145,34 @@
 
         .k-edit-form-container { width: 390px; }
 
-        .k-edit-label { width: auto; }
-        .k-edit-field { width: auto; }
+        .k-edit-label {
+            width: auto;
+            margin-bottom: 0.5rem;
+            label {
+                margin-bottom: 0;
+            }
+        }
+
+        .k-edit-field {
+            width: auto;
+            align-items: center;
+        }
+
+        .k-recur-view {
+            .k-edit-field {
+                .k-numerictextbox {
```

Comment:
There is no need in such nesting of the selectors. Use 
` .k-recur-view .k-edit-field .k-numerictextbox {`